library/build.gradle

```diff
@@ -10,7 +10,7 @@ android {
 
     defaultConfig {
         minSdk 24
-        targetSdk 32
+        targetSdk 33
```

Comment:
This should stay at 32 for now

Reply:
The linting pass trhows the following error with v32:

> /home/runner/work/gpstest/gpstest/library/build.gradle:13: Error: Google Play requires that apps target API level 33 or higher. [ExpiredTargetSdkVersion]

Would you rather prefer to exclude the `ExpiredTargetSdkVersion` at linting level instead of upgrading the target SDK ?

Reply:
Understood, I reverted to the previous v32 and added the linting check in the exclusion list. Just for my own understanding, why is the library and wear modules using the v32 while the GPSTest is using v34 ?

Reply:
That makes sense, thanks for the explanation :)